dnainator-javafx/src/main/resources/fxml/welcome.fxml

```diff
@@ -1,20 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-
+<?import java.lang.String?>
 <?import javafx.scene.control.Button?>
 <?import javafx.scene.control.Label?>
 <?import javafx.scene.control.ListView?>
 <?import javafx.scene.layout.AnchorPane?>
 <?import javafx.scene.text.Font?>
-<AnchorPane xmlns="http://javafx.com/javafx/8.0.45" xmlns:fx="http://javafx.com/fxml/1" fx:controller="nl.tudelft.dnainator.javafx.controllers.WelcomeController">
+<?import javafx.collections.ObservableList?>
+<?import javafx.collections.FXCollections?>
+
+<AnchorPane xmlns="http://javafx.com/javafx/8.0.40" xmlns:fx="http://javafx.com/fxml/1" fx:controller="nl.tudelft.dnainator.javafx.controllers.WelcomeController">
    <children>
-      <ListView fx:id="list" prefHeight="200.0" prefWidth="400.0" AnchorPane.bottomAnchor="100.0" AnchorPane.leftAnchor="100.0" AnchorPane.rightAnchor="100.0" AnchorPane.topAnchor="100.0" />
-        <Button onAction="#startButtonAction" text="Start" AnchorPane.bottomAnchor="20.0" AnchorPane.rightAnchor="20.0" />
+      <ListView fx:id="list" onKeyPressed="#onKeyPressed" onMouseClicked="#onMouseClicked" prefHeight="200.0" prefWidth="400.0" AnchorPane.bottomAnchor="100.0" AnchorPane.leftAnchor="100.0" AnchorPane.rightAnchor="100.0" AnchorPane.topAnchor="100.0">
+        <items>
+          <FXCollections fx:factory="observableArrayList">
+             <String fx:id="selectDB" fx:value="Select a directory containing databases..." />
+          </FXCollections>
+        </items>
+      </ListView>
+      <Button fx:id="loadButton" layoutX="457.0" layoutY="343.0" onAction="#loadButtonAction" text="Load" AnchorPane.bottomAnchor="32.0" AnchorPane.rightAnchor="101.0" />
+        <Button fx:id="deleteButton" disable="true" layoutX="509.0" layoutY="343.0" onAction="#deleteButtonAction" text="Delete" AnchorPane.bottomAnchor="32.0" AnchorPane.rightAnchor="40.0" />
         <Label alignment="CENTER" text="Welcome" textAlignment="CENTER" AnchorPane.leftAnchor="100.0" AnchorPane.rightAnchor="100.0" AnchorPane.topAnchor="20.0">
             <font>
                 <Font size="32.0" />
             </font>
         </Label>
-      <Label alignment="CENTER" text="Please choose a database to open, or press Control+O to open a new file" textAlignment="CENTER" AnchorPane.leftAnchor="100.0" AnchorPane.rightAnchor="100.0" AnchorPane.topAnchor="80.0" />
+      <Label alignment="CENTER" text="Please choose a database to open, or press Ctrl+O to open a new file" textAlignment="CENTER" AnchorPane.leftAnchor="100.0" AnchorPane.rightAnchor="100.0" AnchorPane.topAnchor="80.0" />
```

Comment:
Should this still be rephrased to say "choose a directory with databases", or something along those lines?